_policies/triage-bugzilla.md

```diff
@@ -28,20 +28,19 @@ If you need to change who is responsible for triaging a bug in a component, plea
 
 As a triage owner the queries you should be following for your component are:
 
-* All open bugs in your component without a pending needinfo flag since start of current release cycle
-* All open bugs with pending needinfo flags in your component which have not been modified in two weeks
+* All open bugs of type `defect` in your component without a pending `needinfo` flag or a `stalled` keyword since start of current release cycle which do not have a priorty set
```

Comment:
I expect triage owners to triage all the bugs in their component.  
If a bug is an enhancement it needs a priority set and a target release, if a bug is a task we need to know for what release this is planned. Triage owner should not only triage defects but all the activity in their component. A task such as refactoring fragile code can be risky and we need to know as release managers when these tasks are done wrt to the release cycle, enhancement can lead to release notes and QA needed that we also need to know about. I would probably put an emphasis on defects, but not be exclusive of other types. Maybe something like "All open bugs (prioritizing defects) in your component"